queries/runescript/highlights.scm

```diff
@@ -0,0 +1,92 @@
+; Variables
+[
+  (identifier)
+  (local_variable)
+  (game_variable)
+  (constant_variable)
+  (parameter_variable)
+  (declaration_variable)
+] @variable
+
+; Scripts
+(script
+  trigger: (identifier) @keyword.directive
+  subject: (identifier) @function)
+
+; Calls
+(call
+  [
+    "~"
+    "@"
+  ]? @function
+  callee: (identifier) @function)
+
+; Literals
+((literal) @constant.builtin
+  (#any-of? @constant.builtin "true" "false" "null"))
```

Comment:
shouldn't `true` and `false` be `@boolean`?

Reply:
Good point.